recipes/arrow/all/conanfile.py

```diff
@@ -49,8 +49,9 @@ class ArrowConan(ConanFile):
         "with_glog": ["auto", True, False],
         "with_grpc": ["auto", True, False],
         "with_jemalloc": ["auto", True, False],
-        "with_mimalloc": ["auto", True, False],
```

Comment:
I removed the "auto" value here as there was no real automatic management in place for this options and I didn't investigate the library's build system to know about it. I think it is fine as is.